apps/api/pkg/database/user.go

```diff
@@ -21,3 +22,81 @@ func CreateUser(userId string, email string, firstName string, lastName string)
 
 	return nil
 }
+
+func GetUser(userId string) (models.User, error) {
+	db := GetDB()
+	query := `SELECT id, first_name, last_name, email FROM users WHERE id = $1`
+	row := db.QueryRow(query, userId)
+
+	var user models.User
+	err := row.Scan(&user.UserId, &user.FirstName, &user.LastName, &user.Email)
+	if err == sql.ErrNoRows {
+		return models.User{}, fmt.Errorf("user not found")
+	}
+	if err != nil {
+		return models.User{}, fmt.Errorf("failed to get user from database: %s", err)
+	}
+
+	return user, nil
+}
+
+func CreateTenantUser(user models.User, tenantId int64, role string) error {
+	db := GetDB()
+	query := `INSERT INTO tenantUsers (user_id, tenant_id, role) VALUES ($1, $2, $3)`
+	_, err := db.Exec(query, user.UserId, tenantId, role)
+	if err != nil {
+		return fmt.Errorf("failed to create tenant user: %s", err)
+	}
+
+	return nil
+}
+
+func GetTenantUser(tenantId string) (models.TenantUser, error) {
+	db := GetDB()
+	query := `SELECT user_id, tenant_id, role FROM tenantUsers WHERE user_id = $1`
+	row := db.QueryRow(query, tenantId)
+
+	var tenantUser models.TenantUser
+	err := row.Scan(&tenantUser.UserId, &tenantUser.TenantId, &tenantUser.Role)
+	if err == sql.ErrNoRows {
+		return models.TenantUser{}, fmt.Errorf("tenant user not found")
+	}
+	if err != nil {
+		return models.TenantUser{}, fmt.Errorf("failed to get tenant user from database: %s", err)
+	}
+
+	return tenantUser, nil
+}
```

Comment:
You call it GetTenantUser but it find user with user_id. What do you want to do here?

`SELECT user_id, tenant_id, role FROM tenantUsers WHERE user_id = $1 AND tenant_id=$2` if you want to get TenantUser